spec/tree-view-spec.coffee

```diff
@@ -3019,57 +3019,17 @@ describe "TreeView", ->
         $(workspaceElement).focus()
         expect(treeView.showCurrentFileInFileManager()).toBeUndefined()
 
-    describe "showCurrentFileInFileManager() when some file is opened", ->
-      beforeEach ->
-        filePath = path.join(os.tmpdir(), 'non-project-file.txt')
-        fs.writeFileSync(filePath, 'test')
-        waitsForPromise ->
-          atom.workspace.open(filePath)
+    it "shows file in file manager when some file is opened", ->
+      filePath = path.join(os.tmpdir(), 'non-project-file.txt')
+      fs.writeFileSync(filePath, 'test')
+      waitsForPromise ->
+        atom.workspace.open(filePath)
 
-      it "displays the standard error output when the process fails", ->
-        atom.notifications.clear()
+      runs ->
+        fileManagerProcess = treeView.showCurrentFileInFileManager()
```

Comment:
To avoid actually spawning the file manager, you might want to do something like the following:
```coffee
{BufferedProcess} = require 'atom'
spyOn(BufferedProcess.prototype, 'spawn').andCallFake ->
fileManagerProcess = treeView.showCurrentFileInFileManager()
expect(fileManagerProcess instanceof BufferedProcess).toBeTruthy()
```

I *think* that should work (just make sure it also fails correctly :P).